src/SFA.DAS.EmployerCommitmentsV2.Web/Mappers/SelectProviderConfirmProviderMapper.cs

```diff
@@ -0,0 +1,21 @@
+﻿using SFA.DAS.Commitments.Shared.Interfaces;
+using SFA.DAS.EmployerCommitmentsV2.Web.Models.CreateCohort;
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Mappers
+{
+    public class SelectProviderConfirmProviderMapper : IMapper<ConfirmProviderViewModel, SelectProviderViewModel>
+    {
+        public SelectProviderViewModel Map(ConfirmProviderViewModel source)
+        {
+            return new SelectProviderViewModel
+            {
+                ProviderId = source.ProviderId.ToString(),
+                ReservationId = source.ReservationId,
+                CourseCode = source.CourseCode,
+                AccountHashedId = source.AccountHashedId,
+                EmployerAccountLegalEntityPublicHashedId = source.EmployerAccountLegalEntityPublicHashedId,
```

Comment:
this should be AccountLegalEntityHashedId

Reply:
Fixed in #29